src/test/java/seedu/taassist/logic/commands/AddsCommandTest.java

```diff
@@ -65,6 +77,66 @@ public void execute_validSessionsWithDuplicates_success() throws CommandExceptio
                 commandResult.getFeedbackToUser());
     }
 
+    //==================================== Integration Tests =========================================================
+
+    @BeforeEach
+    public void enterFocusMode() {
+        ModuleClass moduleClassToFocus = model.getModuleClassList().get(0);
+        model.enterFocusMode(moduleClassToFocus);
+        expectedModel.enterFocusMode(moduleClassToFocus);
+    }
+
+    @Test
+    public void execute_addSingleNewSession_success() throws CommandException {
+        ModuleClass focusedClass = model.getFocusedClass();
+        Session validSession = new SessionBuilder().withName(VALID_SESSION_LAB1).build();
+        assert !focusedClass.hasSession(validSession);
+
+        Set<Session> validSessions = new HashSet<>(List.of(validSession));
+        AddsCommand command = new AddsCommand(validSessions);
+        command.execute(expectedModel);
+
+        assertCommandSuccess(command, model, AddsCommand.getCommandMessage(validSessions, new HashSet<>()),
```

Comment:
Should `getCommandMessage` have public visibility? I'd prefer if a hard-coded message is constructed here and the method and other similar methods are set to have private visibility. 

Also if these methods are public, should there be tests for them?

Reply:
@RezwanArefin01 If we hard code the message, then it's going to be quite tedious to properly format the message in test cases and can potentially cause code duplication in many places. So I think the preferred solution is to keep them public and add tests for them (I will do so in another issue).

Also, some commands have `getCommandMessage` while some don't, which is inconsistent. Do you think we should make all commands have/not have the method?

Reply:
Ok, will create new issue regarding tests for `getCommandMessage`